plottable.js

```diff
@@ -6106,6 +6106,9 @@ var Plottable;
                 if (rowPadding == null) {
                     return this._rowPadding;
                 }
+                if (rowPadding === Infinity || rowPadding < 0) {
```

Comment:
what about `NaN` and number strings?
there's a method called `Utils.Math.isValidNumber` that checks for invalid/ non-finite numbers